CONTRIBUTING.md

```diff
@@ -0,0 +1,108 @@
+# Contributing Guidelines
+
+Welcome to Kubernetes. We are excited about the prospect of you joining our [community](https://github.com/kubernetes/community)! The Kubernetes community abides by the CNCF [code of conduct](code-of-conduct.md). Here is an excerpt:
+
+_As contributors and maintainers of this project, and in the interest of fostering an open and welcoming community, we pledge to respect all people who contribute through reporting issues, posting feature requests, updating documentation, submitting pull requests or patches, and other activities._
+
+## Getting Started
+
+### API docs
+
+#### Generate new API docs
+
+1. Update the Makefile for your environment
+  - Set `K8SIOROOT` to the kubernetes/kubernetes.github.io root directory
+    - If you have not already, clone the kubernetes/kubernetes.github.io repo
+  - Set `K8SROOT` to the kubernetes/kubernetes root directory
+    - If you have not already, clone the kubernetes/kubernetes repo
+  - Set `MINOR_VERSION` to the kubernetes/kubernetes minor version (e.g. 8)
+
+2. Copy the `swagger.json` file from Kubernetes.
+  - Go to K8SROOT directory and checkout the `release-<Minor>` branch
+  - Run `make updateapispec` to copy the `swagger.json` for the kubernetes release
+
+3. Run `make api` to build the doc html and javascript (will fail)
+
+4. Update the file `gen_open_api/config.yaml` to fix listed errors
+  - New APIs will be listed as "Orphaned" types, and should be added to the ToC
+  - New versions for existing APIs will be listed as "Orphaned" types, and should replace
+    existing versions in the ToC
+  - **Note:** These should only appear for API types, if field types are listed as
+    Orphaned, then there is an issue with the swagger.json compatibility
+
+5. Run `make api` again after fixing the errors
+
+6. Create a new directory under kubernetes/kubernetes.github.io and
+   update kubernetes/kubernetes.github.io
+  - From kubernetes/kubernetes.io `mkdir docs/api-reference/v1.<Minor>`
+  - Update `docs/reference/index.md` to include the new version
+  - Grep for references to the old version, and update them
+
+6. Run `map copyapi` to copy the files to the kubernetes/kubernetes.github.io directory
```

Comment:
the target directory should be kuberentes/website

Reply:
Fixed :+1: